src/App.js

```diff
@@ -1,12 +1,35 @@
+import { Component } from "react";
+import { BrowserRouter as Router, Switch, Link, Route } from "react-router-dom";
 import "./App.css";
+import Home  from './components/Home';
+import Locations from "./components/Locations";
+import Movies from "./components/Movies";
+import People from "./components/People";
+
 
 function App() {
   return (
-    <div className="app">
-      <main>
-        <h1>Hello, world!</h1>
-      </main>
-    </div>
+    <Router>
+       <Home />
```

Comment:
```suggestion
       <Navbar />
```
It would make more sense to just render the `Navbar` component right here and **not** render the `Navbar` component in the `Home` component or anywhere else.